code/modules/clothing/outfits/syndicate.dm

```diff
@@ -0,0 +1,826 @@
+//top outfit of everything syndicate. Don't change this.
+
+/datum/outfit/job/syndicate
+	name = "Syndicate (mostly) Empty"
+
+	uniform = /obj/item/clothing/under/syndicate
+	box = /obj/item/storage/box/survival/syndie
+	id = /obj/item/card/id/syndicate_command/crew_id
+
+	backpack = /obj/item/storage/backpack/security
+	satchel = /obj/item/storage/backpack/satchel/sec
+	duffelbag = /obj/item/storage/backpack/duffelbag/syndie
+	courierbag = /obj/item/storage/backpack/messenger/sec
+
+/datum/outfit/job/syndicate/post_equip(mob/living/carbon/human/H, visualsOnly)
+	. = ..()
+	if(visualsOnly)
+		return
+	H.faction |= list("PlayerSyndicate")
+
+
+//generates a codename and assigns syndicate access, used in the twinkleshine.
+/datum/outfit/job/syndicate/proc/assign_codename(mob/living/carbon/human/H)
+	var/obj/item/card/id/I = H.wear_id
+	I.registered_name = pick(GLOB.twinkle_names) // squidquest real
```

Comment:
```suggestion
	I.registered_name = pick(GLOB.twinkle_names) + "-" + num2text(rand(1, 12)) // squidquest real
```
otherwise people are going to be named "Crimson", and there's a slim chance you might get two people with the same shade of red. or maybe even more